src/lib/SapphireClient.ts

```diff
@@ -33,6 +39,7 @@ export class SapphireClient extends Client {
 		super(options);
 
 		this.clientID = options.id ?? null;
```

Comment:
Unrelated to this PR but Vlad will do what Vlad does best. Can this be also set on ready to the user? I understand this would be needed for..stateless(?), but for most users it should be auto set

Reply:
It's out of the PR's scope to add this event, but sure.

Reply:
Also...

```typescript
× prettier --write:
[error] src\events\CoreReady.ts: SyntaxError: Expression expected. (12:20)
[error]   10 |  public run() {
[error]   11 |          // eslint-disable-next-line prettier/prettier
[error] > 12 |          this.client.id ??= this.client.user?.id ?? null;
[error]      |                           ^
[error]   13 |  }
[error]   14 | }
[error]   15 |
```

Reply:
you should make your commit hooks for ts use eslint, not prettier directly..